iac-extensions/arm/src/test/resources/checks/ResourceSpecificAdminAccountCheck/Microsoft.ContainerRegistry_registries.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "name": "Noncompliant: adminUserEnabled is set to true",
+      "type": "Microsoft.ContainerRegistry/registries",
+      "apiVersion": "2019-05-01",
+      "properties": {
+        "adminUserEnabled": true
+      }
+    },
+
```

Comment:
Nitpick, empty line

Reply:
I usually add this empty line to separate between NonCompliant and Compliant test cases :smile: